src/app/features/request/new/new.component.html

```diff
@@ -1,5 +1,169 @@
 <app-page>
   <ng-container content>
+    <div class="header">
+      <div class="container md:container md:mx-auto p-16 pt-44">
```

Comment:
"md:container" class is useless in case "container" exists

remove "md:container"